NetSDK/Services/Client/Classes/ConfigurationOptions.cs

```diff
@@ -8,5 +8,27 @@ namespace Splitio.Services.Client.Classes
     public class ConfigurationOptions
     {
         public string LocalhostFilePath { get; set; }
+        public int? FeaturesRefreshRate { get; set; } 
+        public int? SegmentsRefreshRate { get; set; } 
+        public bool RandomizeRefreshRates { get; set; } 
+        public int? ImpressionsRefreshRate { get; set; }
+        public int? MaxImpressionsLogSize { get; set; }  
+        public long? ConnectionTimeout { get; set; } 
+        public long? ReadTimeout { get; set; } 
+        public int? Ready { get; set; }  
+        public int? MaxMetricsCountCallsBeforeFlush { get; set; } 
+        public int? MaxMetricsTimeBetweenCallsBeforeFlush { get; set; } 
+        public int? SplitsStorageConcurrencyLevel { get; set; } 
+        public string SdkMachineName { get; set; }
+        public string SdkMachineIP{ get; set; } 
+
+
+        /*
+    <add key="SPLIT_CHANGES_URL_TEMPLATE" value="splitChanges" />
+    <add key="URL_PARAMETER_SINCE" value="?since=" />
+    <add key="SEGMENT_CHANGES_URL_TEMPLATE" value="segmentChanges/{segment_name}" />
+    <add key="TEST_IMPRESSIONS_URL_TEMPLATE" value="testImpressions/bulk" />
+    <add key="METRICS_URL_TEMPLATE " value="metrics/{endpoint}" />
+         * */
```

Comment:
removing the xml configuration file we would need to recompile the library if we want to add some change
What do you think if we can have the bet of both approaches?
1 - by default without a xml file we can have defualt values
2-  with an xml this will override the default values